setup.py

```diff
@@ -0,0 +1,104 @@
+import os
```

Comment:
Are the classes in this file (e.g. BazelExtension) taken or adapted from some third-party library? If so, maybe point to the repository and commit, and keep the license header. Otherwise add our own license header.